### PR TITLE
CFM-1955 Re-add nifi_scaling hostGroup to 7.2.10 templates. Added nif…

### PR DESCRIPTION
**Pull request by vbrodetskyi**
…i scaling hostGroup and also ssl properties for light duty cluster because CM still not fixed that on their side.

See detailed description in the commit message.

### CODE REVIEW

**Review by lajosrodek — APPROVED** [review]
LGTM

**Review by adenes — APPROVED** [review]
Thanks @vbrodetskyi , LGTM

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp [hunk 1]
```diff
@@ -48,6 +48,30 @@
             "roleType": "NIFI_NODE",
             "base": true,
             "configs": [
+              {
+                "name": "ssl_enabled",
+                "value": "true"
+              },
+              {
+                "name": "ssl_server_keystore_location",
+                "value": "{{CM_AUTO_TLS}}"
```
Comment (bergerdenes) [inline core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:57]: sure?

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp [hunk 2]
```diff
@@ -48,6 +48,30 @@
...
+              },
+              {
+                "name": "ssl_server_keystore_password",
+                "value": "{{CM_AUTO_TLS}}"
```
Comment (bergerdenes) [inline core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:61]: sure?

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp [hunk 3]
```diff
@@ -48,6 +48,30 @@
...
+              },
+              {
+                "name": "ssl_server_keystore_keypassword",
+                "value": "{{CM_AUTO_TLS}}"
```
Comment (bergerdenes) [inline core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:65]: sure?

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp [hunk 4]
```diff
@@ -48,6 +48,30 @@
...
+              },
+              {
+                "name": "ssl_client_truststore_location",
+                "value": "{{CM_AUTO_TLS}}"
```
Comment (bergerdenes) [inline core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:69]: sure?

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp [hunk 5]
```diff
@@ -48,6 +48,30 @@
...
+              },
+              {
+                "name": "ssl_client_truststore_password",
+                "value": "{{CM_AUTO_TLS}}"
```
Comment (bergerdenes) [inline core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:73]: sure?

**Review by bergerdenes — CHANGES_REQUESTED** [review]
please answer the question of why the value for several properties is the same handlebars variable

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp (hunk 1 above)
Comment (vbrodetskyi) [reply core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:57]: This value {{CM_AUTO_TLS}} is not a real value, but a flag for CM to set this property value automatically. These property/value pairs were added/tested for 7.2.8 and 7.2.9 and everything works fine, as expected.

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp (hunk 2 above)
Comment (vbrodetskyi) [reply core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:61]: This value {{CM_AUTO_TLS}} is not a real value, but a flag for CM to set this property value automatically. These property/value pairs were added/tested for 7.2.8 and 7.2.9 and everything works fine, as expected.

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp (hunk 3 above)
Comment (vbrodetskyi) [reply core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:65]: This value {{CM_AUTO_TLS}} is not a real value, but a flag for CM to set this property value automatically. These property/value pairs were added/tested for 7.2.8 and 7.2.9 and everything works fine, as expected.

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp (hunk 4 above)
Comment (vbrodetskyi) [reply core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:69]: This value {{CM_AUTO_TLS}} is not a real value, but a flag for CM to set this property value automatically. These property/value pairs were added/tested for 7.2.8 and 7.2.9 and everything works fine, as expected.

File: core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp (hunk 5 above)
Comment (vbrodetskyi) [reply core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp:73]: This value {{CM_AUTO_TLS}} is not a real value, but a flag for CM to set this property value automatically. These property/value pairs were added/tested for 7.2.8 and 7.2.9 and everything works fine, as expected.

**Review by bergerdenes — APPROVED** [review]

**Pull request merged**